.vscode/settings.json

```
@@ -51,5 +51,6 @@
     "cSpell.enableFiletypes": [
         "cuda"
     ],
-    "C_Cpp.default.configurationProvider": "ms-vscode.cmake-tools"
+    "C_Cpp.default.configurationProvider": "ms-vscode.cmake-tools",
+    "python.pythonPath": "/usr/bin/python3"
```

```suggestion
    "C_Cpp.default.configurationProvider": "ms-vscode.cmake-tools"
```
Not all systems install python in this location.

---

Sorry, didn't notice this from when I took over the PR.